### DISCUSSION TITLE
Python

### DISCUSSION THREAD

**Discussion by Youknowwhoitis970** [ish-app/ish#1685] [Q&A]
I'm trying to sqlmap a website but each time it says "env: can't execute 'python': No such file or directory". Can someone explain? Or how to fix it because I've tried everything

**Comment by kris-hansen** [comment]
Once you've installed python-3 with 
```
apk add py3
```
You will have a /bin/python3 binary installed and you may want to symlink to /bin/python with
```
ln -s /bin/python3 /bin/python
```
This will give you a python binary in your path.
